Data/sync/BraveCryptoJSFunctions.swift

```diff
@@ -0,0 +1,41 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveShared
+
+struct BraveCryptoJSFunctions: BrowserifyExposable, BundledJSProtocol {
+    let bundle: Bundle
+    let name: String
+    let exposedFunctions: [ExposedJSFunction]
+    
+    struct Names {
+        static let fromBytesOrHex = "fromBytesOrHex"
+        static let toBytes32 = "toBytes32"
+    }
+    
+    init?() {
+        name = "crypto"
+        bundle = Bundle.data
+        
+        exposedFunctions = [ExposedJSFunction(name: Names.fromBytesOrHex, body: Bodies.fromBytesOrHex),
+                            ExposedJSFunction(name: Names.toBytes32, body: Bodies.toBytes32)]
+    }
+    
+    private struct Bodies {
+        static let fromBytesOrHex =
+        """
+        function(bytes){
+           return module.exports.passphrase.fromBytesOrHex(bytes);
```

Comment:
I think we should use the `Names.rawValue` for the function calling here:

```suggestion
           return module.exports.passphrase.\(Names.fromBytesOrHex)(bytes);
```

Reply:
This function still should be copied 1:1 from javascript file.
For me it's easier if you can simply copy paste it into the string,.
I will change it to your suggestion